### PR TITLE
Code examples for episode 433

### PR DESCRIPTION
**Pull request by int3l**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -45,3 +45,4 @@ per-file-ignores =
     */ep407/rev0*/a/main.py:E402
     */ep418/rev0*/t.py:F821
     */ep430/rev0*/t.py:F821
+    */ep433/rev0*/t.py:E999
```
Comment (int3l) [inline setup.cfg:48]: This one is a gross, but I guess that it's needed 😂

Comment (asottile) [reply setup.cfg:48]: wait what's E999 O_o

Comment (asottile) [reply setup.cfg:48]: oh invalid syntax, right

**Review by asottile — APPROVED** [review]

**Pull request merged**